packages/superfluidDaoFront/src/components/CardDaos.tsx

```diff
@@ -0,0 +1,26 @@
+import { Link } from "react-router-dom";
+
+type Props = {
+  name: string;
+  description: string;
+  link: string;
+};
+
+const CardDaos = (props: Props) => {
```

Comment:
```suggestion
const CardDaos = ({ name, description, link }: Props) => {
```
d'habitude on destructure les props pour pas avoir a ecrire `props. ....` partout